superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -107,6 +106,9 @@ const TitleLink = styled.span`
 
 const TitleRight = styled.span`
   margin-left: ${({ theme }) => theme.gridUnit * 2}px;
```

Comment:
```suggestion
```
If we're absolutely positioned based on `right` coordinates, this `margin-left` probably isn't doing anything useful